internal/server/data/migrations.go

```diff
@@ -40,8 +40,7 @@ func migrations() []*migrator.Migration {
 		{
 			ID: "202206081027",
 			Migrate: func(tx *gorm.DB) error {
-				_ = tx.Migrator().DropConstraint(&models.Group{}, "idx_groups_name_provider_id")
-				return nil
+				return tx.Migrator().DropIndex(&models.Group{}, "idx_groups_name_provider_id")
```

Comment:
This was a bug in one of our existing migrations. I guess we never noticed because we ignored the error.

I fixed it in place. This is an old enough migration that I guess not many people will have this old index. 

We could also create a new migration to ensure that it gets removed.

Maybe I should add `HasIndex` here.